docs/content/integrations/dbt/reference.mdx

```diff
@@ -188,8 +247,14 @@ If you are managing your Dagster project in the same git repository as your dbt
 In your CI/CD workflows for your Dagster and dbt project:
 
 1. Include any secrets that are required by your dbt project in your CI/CD environment.
-2. Run `dbt deps` to build your dbt project's dependencies.
-3. Run `dbt parse` to create a dbt manifest for your Dagster project.
+3. Depending on whether [DbtProject](/\_apidocs/libraries/dagster-dbt#dagster_dbt.DbtProject) is used, this step will vary:
```

Comment:
It looks like this section and the one that starts on line 226 are the same - my comments from there apply here too